src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -722,7 +722,7 @@ public string AstIdentifierGuid
         {
             get
             {
-                return GUID.ToString().Replace("-", string.Empty).ToLower();
+                return GUID.ToString("N");
```

Comment:
The default option is "D" if no Parma passed in which is the format with dashes. "N" is the one for format without dashes and in lowercase. This is doing what that code does so I just replaced the these code.